code/Gamemode.cs

```diff
@@ -20,6 +20,8 @@ public Gamemode()
 
         public static Gamemode Instance { get; private set; }
 
+        public event Action<MainPlayer> PlayerAddedEvent;
```

Comment:
Is it not EvenHandler we should use ?

Reply:
What event handler ?

Reply:
Event is designed with EventHandler in mind

https://stackoverflow.com/questions/1431359/event-action-vs-event-eventhandler
https://docs.microsoft.com/fr-fr/dotnet/api/system.eventhandler?view=net-5.0

Reply:
I wanted to follow sbox coding with their event. But I didn't know about eventhandler ! Do you want me to change ?

Reply:
Yep, the convention seems to be EventHandler. I don't know why facepunch didn't follow it lmao